### PR TITLE
Yarn

### PR DESCRIPTION
**Pull request by steveoh**
i think this makes getting started simpler. 

I added a readme to say what I did.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### One timers
+
+- npm i -g yarn
+- yarn config set workspaces-experimental true
+
+### Install
+
+- git clone
+- lerna bootstrap
```
Comment (stdavis) [inline README.md]: `lerna bootstrap` did not install any of the root-level dependencies for me.

Comment (stdavis) [reply README.md]: Should this be changed to `yarn` which runs lerna bootstrap and grunt babel & sass automatically?

Comment (stdavis) [reply README.md]: Nevermind. I was on a different branch. I had a local branch with the same name as this one. Please disregard.

**Comment by stdavis** [comment]
This doesn't work if you have two different version of the same package dependency. Here's what I did to test:
1. `lerna clean && rm -rf node_modules/`
1. Set both packages to esri 4.6.x
1. `lerna bootstrap` - everything was working correctly
1. Set map-tools to esri 4.6.x and mouse-trap to 4.7.x
1. `lerna bootstrap`

Issues that I found:
- mouse-trap had it's own node_modules folder with the esri deps in it which obviously doesn't work with the jasmine spec runner paths
- root/node_modules/arcgis-js-api has a nest node_modules folder which breaks the dojo build system. I think that it would be wise to set up a dojo build for this project to make sure that we aren't breaking a build when me make improvements.

I think that we should work on a solution that keeps all of the dev dependencies at the root level but all of the front-end dependencies within the packages. That way the packages can be version independent in their dependencies. I _believe_ that the current map-tools branch does this.

**Comment by steveoh** [comment]
I agree that is an issue.

**Pull request closed without merging**